biome.json

```diff
@@ -1,12 +1,45 @@
 {
   "$schema": "https://biomejs.dev/schemas/1.9.4/schema.json",
-  "formatter": {
-    "enabled": false
+  "vcs": {
+    "enabled": false,
+    "clientKind": "git",
+    "useIgnoreFile": false
   },
-  "linter": {
-    "enabled": false
+  "files": {
+    "ignoreUnknown": false,
+    "ignore": []
+  },
+  "formatter": {
+    "enabled": true,
+    "indentStyle": "tab"
   },
   "organizeImports": {
     "enabled": true
+  },
+  "linter": {
+    "enabled": true,
+    "rules": {
+      "recommended": true,
+      "complexity": {
+        "noForEach": "warn"
+      },
+      "style": {
+        "noNonNullAssertion": "off",
```

Comment:
This one can be turned to `warn` once it works better to disable lines or files in biome v2: https://github.com/biomejs/biome/issues/4569.

Until then I'd recommend to keep lines like `// eslint-disable-next-line @typescript-eslint/no-explicit-any` in the project to be able to simple find/replace later one.

Let me know what you think? @0xonramp @tokdaniel